crowdship-app/app/deliveryrequest.tsx

```diff
@@ -8,24 +8,36 @@ import {
   TouchableOpacity,
   TextInput,
   ActivityIndicator,
+  Modal,
+  Alert,
 } from "react-native";
 import * as ImagePicker from "expo-image-picker";
-import { AntDesign } from "@expo/vector-icons";
-import { Feather } from "@expo/vector-icons";
+import { AntDesign, Feather, MaterialIcons } from "@expo/vector-icons";
```

Comment:
We should only be using one icon library. See if AntDesign has the icon first, if not, use Feather. The names for the icons are not intuitive, so you might just have to look through the library manually.